### ISSUE TITLE
Colour picker tool

### ISSUE THREAD

**Issue by hectorbennett** [hectorbennett/rust-wasm-image-editor#23]

**Comment by Atharshoyeb** [comment]
I am a beginner in open source can u explain the issue I would like to work on this?

**Comment by hectorbennett** [comment]
Hi Athar!

Welcome to the project! I've only recently started making this so I haven't been able to properly document things yet - but it would be great if you did want to become a contributor!

Basically the first step would be to get it running locally if you haven't already. I need to properly document this - but the main steps are

1. Install node/npm
2. Install rust
3. Install wasm-pack
4. then cd to the project folder and run 'npm ci' followed by 'npm run dev'.

I think actually this task might not be the best one to start with - as I had already started working on this one myself (https://github.com/hectorbennett/rust-wasm-image-editor/tree/23-colour-picker-tool)

This one should be a good one to start with, and hopefully shouldn't take long:

https://github.com/hectorbennett/rust-wasm-image-editor/issues/14

I can write a proper description in that issue of what's needed
